docs/source/overview/quicktour.md

```diff
@@ -1,42 +1,47 @@
 (Quick-tour)=
 # Quick Tour
 
-Here is a quick guide on using Olive for model optimization with two steps. We will focus on accelerating a PyTorch model on the CPU leveraging ONNX conversion and ONNX quantization.
+Here is a quick guide on using Olive for model optimization. We will focus on accelerating a PyTorch model on the CPU. It is a simple three step process.
+
+1. **Install Olive and necessary packages.**
+
+You can install olive using pip install.
 
-## Step1 Install Olive
-Before you begin, install Olive and the necessary packages.
 ```bash
 pip install olive-ai
 ```
-Refer to the [Installation](Installation) section for more details.
 
-## Step 2 Run Olive with Model Optimization Workflow
-Olive model optimization workflows are defined using config JSON files. You can use the Olive CLI to run the pipeline.
+2. **Describe your model and your needs in a json configuration file. This will be the input to the Olive.**
 
-First, install required packages according to your config file.
-```
-python -m olive.workflows.run --config user_provided_info.json --setup
-```
-Then, optimize the model
+Olive needs information about your model. For example, how to load the model, name and shape of input tensors. You can also select the target hardware and list of optimizations you want to perform on the model. You can provide this information in a json file as an input to the Olive.
+
+3. **Accelerate the model using Olive.**
+
+The last step is the simplest one. You just need to run following simple command.
 ```
-python -m olive.workflows.run --config user_provided_info.json
+python -m olive.workflows.run --config my_model_acceleration_description.json
 ```
-
 or in python code:
-
 ```python
 from olive.workflows import run as olive_run
-olive_run("user_provided_info.json")
+olive_run("my_model_acceleration_description.json")
 ```
 
 ```{note}
 `olive.workflows.run` in python code also accepts python dictionary equivalent of the config JSON object.
+
+You can use setup mode `python -m olive.workflows.run --config my_model_acceleration_description.json --setup` to identify list of additional packages you may need to install for your workflow.
 ```
 
-Now, let's take a look at the json configuration file you need to provide to optimize your model.
+## Information needed to accelerate a model
+
+Typically, you need to have input model information such as model type, input names and shapes, where the model is stored.  You would also know your desired performance requirements in terms of Latency, Accuracy etc. Along with this information you need to provide Olive list of model transformations and optimizations you want to apply. Optionally you can also select target hardware and select additional Olive features. Now, let's take a look at how you can include this information in the json configuration file you will use as an Olive input.
```

Comment:
Two blanks before `You would also know....`

Reply:
Good catch!